packages/accessible-react-app/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "simple-react-boilerplate",
+  "version": "1.0.0",
+  "description": "A simple React boilerplate to spin up further experiments",
+  "main": "index.js",
+  "author": "Trevor Pierce",
+  "license": "CC0-1.0",
+  "scripts": {
+    "build": "npm-run-all clean webpackBuild server",
+    "clean": "rm -rf dist/",
```

Comment:
I removed the lint command from this app. Since we aren't running `lerna bootstrap` on this package, that means none of the node_modules are installed. This causes linting issues with unresolvable require statements. I figure we can add it back once we figure out a way to get this package to install on linux.